tests/validation/cattlevalidationtest/core/common_fixtures.py

```diff
@@ -1625,3 +1629,35 @@ def get_side_kick_container(super_client, container, service, service_name):
     secondary_con = get_service_container_with_label(
         super_client, service, service_name, label)
     return secondary_con
+
+
+def validate_internal_lb(super_client, lb_service, services,
+                         host, con_port, lb_port):
+    # Access each of the LB Agent from the client container
+    lb_containers = get_service_container_list(super_client, lb_service)
+    assert len(lb_containers) == lb_service.scale
+    for lb_con in lb_containers:
+        lb_ip = lb_con.primaryIpAddress
+        target_count = 0
+        for service in services:
+            target_count = target_count + service.scale
+        expected_lb_response = get_container_names_list(super_client,
+                                                        services)
+        assert len(expected_lb_response) == target_count
+        # Validate port mapping
+        ssh = paramiko.SSHClient()
+        ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+        ssh.connect(host.ipAddresses()[0].address, username="root",
+                    password="root", port=int(con_port))
+
+        # Validate lb service from this container using LB agent's ip address
```

Comment:
as a part of this test, we should execute one negative test - try to access the LB via host ip address, and make sure that you get the error back.